packages/arb-token-bridge-ui/src/__test__/tokenLists.test.ts

```diff
@@ -0,0 +1,24 @@
+import { BRIDGE_TOKEN_LISTS, fetchTokenListFromURL } from '../tokenLists'
+
+/**
+ * TODO: Move this test (or a similar one) to the token lists repo
+ */
+describe('Token Lists', () => {
+  jest.setTimeout(20_000)
+
+  it('validates that all token lists have the proper schema', async () => {
+    const remoteTokenListUrls = BRIDGE_TOKEN_LISTS
+      //
```

Comment:
missing comment? :D

Reply:
It's for formatting, I think it looks cleaner when you have each step in a separate line you're chaining mapping / filtering over an array